api/qr.py

```diff
@@ -0,0 +1,15 @@
+def make(update, context):
+    """ Command to generate QR code from given data"""
+    message = update.message
+    data = ' '.join(context.args)
+
+    if not data:
+        message.reply_text(
+            text="*Usage:* `/qr {CONTENT}`\n"
+                 "*Example:* `/qr superserio.us`"
+        )
+
+    else:
+        message.reply_photo(
+            photo=f'https://api.qrserver.com/v1/create-qr-code/?data={data}',
```

Comment:
We should encode the data using `urllib.parse.urlencode` since this is entirely user input.
Also could you change this behaviour in other modules like currency, goodreads, ud etc wherever we're manually dealing with urls.
Wherever we're using `requests.get` we can pass the query params in the `params` arg to do this.

Reply:
I'll make the change for this PR. For everything else a new PR would be more appropriate.